ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -20,8 +36,11 @@ <h4 class="header-title m-t-0 m-b-20">Speaking Practice</h4>
   
           <div class="sec ec card-body show-expression inline-block  text-center align-middle" id="expression-container"> <!-- card for showing expression -->
             <h1 id='expression-text' class="card-title text-center show-expression"></h1>
-            <h2 id="timer-text" class="show-expression"></h2>
-            <div class="timer bg-muted" style="height: 100px; width: 100%">
+            <div>
+              <h2 id="timer-text" class="show-expression" style="display: inline-block;"></h2>
+              <img id="curr-icon" class=""  style="display: none;" src="../../../../static/ComSemApp/images/reading_icon.png" width="35" height="35">
```

Comment:
Small thing. I would put a `<br>` between the text and the icon so they're stacked rather than beside one another.